google-cloud-language/pom.xml

```diff
@@ -97,10 +97,10 @@
           <execution>
             <phase>generate-resources</phase>
             <goals>
-                <goal>write-project-properties</goal>
+              <goal>write-project-properties</goal>
             </goals>
             <configuration>
-                <outputFile>${project.build.outputDirectory}/project.properties</outputFile>
+              <outputFile>${project.build.outputDirectory}/properties/${project.artifactId}/project.properties</outputFile>
```

Comment:
Right now it reports beta version. I am actually a bit confused about this. Should the gapic version always be alpha?